content/streamlit-cloud/get-started/manage-your-app.md

```diff
@@ -176,3 +177,82 @@ To unfavorite an app, click the star (⭐) again.
 Click [here](/knowledge-base/deploy/upgrade-streamlit-version-on-streamlit-cloud) to learn more about upgrading the Streamlit version of your app on Streamlit Cloud.
 
 </Tip>
+
+## Analytics Modal
+
+Once you start or are invited to a Streamlit workspace, you have access to 2 types of analytics:
+
+1. [Workspace analytics](/streamlit-cloud/get-started/manage-your-app#workspace-analytics): shows you how many viewers in total have visited _all_ the apps in your workspace.
+   <Image alt="Workspace analytics" src="/images/streamlit-cloud/workspace-analytics.gif" />
+2. [App viewers](/streamlit-cloud/get-started/manage-your-app#app-viewers): shows you who has recently viewed your workspace’s individual apps and when.
+   <Image alt="Workspace analytics" src="/images/streamlit-cloud/app-viewers-data.gif" />
+
+<Note>
+
+The Analytics Modal is visible to everyone with access to your workspace, including admins, developers, or anyone with viewer access to a workspace.
+
+</Note>
+
+### Workspace analytics
+
+Streamlit Cloud enables you to view analytics data for all apps in your workspace in one central dashboard. At a glance, you get an overview of how active your workspace is and how popular your apps are.
+
+Say you're the Head of Data and want to see if the Streamlit usage in your company has increased over time. Or say you're an open-source developer and want to see how popular your apps have become. The Workspace analytics dashboard is your one-stop-shop for these insights.
```

Comment:
I'm not sure this adds anything to the docs, since I would think it's pretty self-evident why someone would want to know if people are using their apps

Reply:
> Say you're the Head of Data and want to see if the Streamlit usage in your company has increased over time. Or say you're an open-source developer and want to see how popular your apps have become. The Workspace analytics dashboard is your one-stop-shop for these insights.

I'm inclined to agree with you.. Seems this paragraph is more suited to the accompanying blog post. I'll sync with Diana to see if there's any reason to keep this included. If not, I'll remove it.